src/utils.ts

```diff
@@ -1,5 +1,21 @@
 import { useLocation } from 'react-router';
+import { parse, format } from 'date-fns';
+import { ko } from 'date-fns/locale';
 
 export function useQuery() {
   return new URLSearchParams(useLocation().search);
 }
+
+export function changeStringToDate(dateString: string) {
+  const date = parse(dateString, 'yyyy-MM-dd', new Date());
+  return date;
+}
+
+export function makeDateFormat(date: Date) {
+  const result = format(date, 'M.ddeee', { locale: ko });
+  return result;
+}
+
+export function numberWithCommas(price: number) {
+  return price.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+}
```

Comment:
함수 이름은 `numberWithComma`로 꽤 범용적인 이름을 가지고 있는데, 인자가 `price`인 것이 어색한 것 같습니다!

그리고 이 동작은 정규식을 쓰지 않고 `Number#toLocaleString` 메소드를 사용해도 될 것 같슴다! 🙏

```suggestion
export function numberWithCommas(number: number) {
  return number.toLocaleString();
}
```